src/main/java/com/amazon/ion/impl/IonWriter_1_1.kt

```diff
@@ -60,10 +60,10 @@ interface IonWriter_1_1 {
     fun writeAnnotations(annotation0: Int, annotation1: Int)
 
     /**
-     * Writes three or more annotations for the next value.
+     * Writes any number of annotations for the next value.
      * [writeAnnotations] may be called more than once to build up a list of annotations.
      */
-    fun writeAnnotations(annotation0: Int, annotation1: Int, vararg annotations: Int)
+    fun writeAnnotations(annotations: IntArray)
```

Comment:
🗺️ I realized that it (a) made the implementation cleaner and (b) just made sense to be explicit about the array here, so I changed the function signature.

Reply:
Does this ever need to be called from Java? What does that look like?

Reply:
For now, I don't think the Ion 1.1 raw writer needs to be called from Java. However, it is still callable from Java. `IntArray` is Kotlin's name for `int[]`.

As for removing the vararg parameter—instead of this:
```java
writer.writeAnnotations(1, 2, 3, 4, 5);
```
Now you have to be explicit about the array:
```java
write.writeAnnotations(new int[]{1, 2, 3, 4, 5})
```

This is a reversible decision.